vmdb/app/presenters/menu/polymorph_item.rb

```diff
@@ -0,0 +1,7 @@
+module Menu
+  class PolymorphItem < Item
+    def name
+      super.call
```

Comment:
Why the `.call`?  Also, what purpose does this class serve?  I'm concerned you're using inheritance for code sharing instead of for a specialization.